Task_TheGridProblem/index.css

```diff
@@ -1 +1,7 @@
-/* Write code below for GRID PROBLEM TASK */
+/* Write code below for GRID PROBLEM TASK */
+.dd{
+display: grid;
+gap: 1rem;
+grid-template-columns: 1fr 1fr 1fr;

```

Comment:
Similar to the previous line, each 'grid-template-rows' value should have a unit of measurement for better clarity.